### ISSUE TITLE
Error in REACH processor handling polarity in simple nested events

### ISSUE THREAD

**Issue by johnbachman** [sorgerlab/indra#337]
Upon reflection, I suspect this may be a known bug, but if it's not already logged somewhere I thought it worth documenting. I noticed the following problematic issue with a sentence I came across:

```
In [1]: from indra.sources import reach

In [2]: rp = reach.process_text('BYL719, however, effectively inhibited Akt phosph
   ...: orylation, alone or combined with lapatinib.')

In [3]: rp.statements
Out[3]: [Phosphorylation(BYL719(), AKT())]

In [5]: rp.statements[0].evidence
Out[5]: [Evidence(reach, api2256, {'found_by': 'Phosphorylation_syntax_4_noun'}, BYL719, however, effectively inhibited Akt phosphorylation, alone or combined with lapatinib.)]
```

Clearly this should be some kind of negative regulation or dephosphorylation event, not a phosphorylation event. I thought this was a REACH bug, but then I looked in `reach_output.json` and saw something strange. There are two frames in the JSON, shown below:

``` 
"events":{
    "frames":[
      {
        "frame-id":"evem-api2256-UAZ-r1-Reach-0-23071",
        "text":"Akt phosphorylation",
        "arguments":[
          {
            "text":"Akt",
            "argument-type":"entity",
            "type":"theme",
            "object-type":"argument",
            "index":0,
            "arg":"ment-api2256-UAZ-r1-Reach-0-180695"
          }
        ],
        "type":"protein-modification",
        "frame-type":"event-mention",
        "subtype":"phosphorylation",
        "is-direct":false,
        "end-pos":{
          "reference":"pass-api2256-UAZ-r1-Reach",
          "offset":58,
          "object-type":"relative-pos"
        },
        "trigger":"phosphorylation",
        "object-type":"frame",
        "start-pos":{
          "reference":"pass-api2256-UAZ-r1-Reach",
          "offset":39,
          "object-type":"relative-pos"
        },
        "sentence":"sent-api2256-UAZ-r1-Reach-0",
        "found-by":"Phosphorylation_syntax_4_noun",
        "verbose-text":"BYL719, however, effectively inhibited Akt phosphorylation, alone or combined with lapatinib."
      },
      {
        "frame-id":"evem-api2256-UAZ-r1-Reach-0-23070",
        "text":"BYL719, however, effectively inhibited Akt phosphorylation",
        "arguments":[
          {
            "text":"Akt phosphorylation",
            "argument-type":"event",
            "type":"controlled",
            "object-type":"argument",
            "index":0,
            "arg":"evem-api2256-UAZ-r1-Reach-0-23071"
          },
          {
            "text":"BYL719",
            "argument-type":"entity",
            "type":"controller",
            "object-type":"argument",
            "index":0,
            "arg":"ment-api2256-UAZ-r1-Reach-0-180696"
          }
        ],
        "type":"regulation",
        "frame-type":"event-mention",
        "subtype":"negative-regulation",
        "is-direct":false,
        "end-pos":{
          "reference":"pass-api2256-UAZ-r1-Reach",
          "offset":58,
          "object-type":"relative-pos"
        },
        "trigger":"inhibited",
        "object-type":"frame",
        "start-pos":{
          "reference":"pass-api2256-UAZ-r1-Reach",
          "offset":0,
          "object-type":"relative-pos"
        },
        "sentence":"sent-api2256-UAZ-r1-Reach-0",
        "found-by":"Negative_regulation_syntax_1_verb",
        "verbose-text":"BYL719, however, effectively inhibited Akt phosphorylation, alone or combined with lapatinib."
      }
    ],
```

There are two frames, one which lists an AKT phosphorylation event, but does not list BYL719 as the controller. The other is a "regulation" event, with subtype "negative-regulation", that lists BYL719 controller, and the AKT phosphorylation event itself as the controlled.

The REACH processor pulls out the Phosphorylation event, tagging the evidence with the `Phosphorylation_syntax_4_noun` rule. However, it also lists BYL719 as the controller of that event, which it is not. In addition, it does not yield the negative regulation event in any form.

While handling nested events in the general case is non-trivial, it seems that for sentences like these we would want to turn this into an indirect Dephosphorylation event. Perhaps the issue is simply that the event subtype "negative-regulation" is being missed here--after all, if this was a positive regulation of the Phosphorylation event, this would be deemed a correct extraction.

**Comment by bgyori** [comment]
Oh I see... This is a pretty basic problem with the Processor, and I'm really surprised we didn't notice it so far!

**Comment by bgyori** [comment]
Fixed in d6b45e4dbdbcf76a4ede4928e7004e17502a4b9c
